src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -27,12 +27,17 @@ private abstract class AbstractTableDataSourceFindUsagesContext :
         {
             private readonly CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
 
-            private ITableDataSink _tableDataSink;
+            private ImmutableList<ITableDataSink> _tableDataSinks = ImmutableList<ITableDataSink>.Empty;
 
             public readonly StreamingFindUsagesPresenter Presenter;
             private readonly IFindAllReferencesWindow _findReferencesWindow;
             protected readonly IWpfTableControl2 TableControl;
 
+            /// <summary>
+            /// Indicates the last stable state communicated with sinks.
+            /// </summary>
+            private bool isStable;
```

Comment:
private fields should be prefaced with `_`.

Reply:
Oh right. done.

Reply:
it's also not clear how this is ever read... did i miss that somewhere?